tutorials/migration.md

```diff
@@ -292,134 +237,112 @@ To get a list of all currently installed items run:
-installing persistence add-ons.
+### PaperUI Approach (Alternative)
 
-### PaperUI Approach
+These instructions assume no edits have been made to `runtime.cfg` or `addons.cfg` as described in the previous sections.
 
-These instructions assume no edits have been made to `runtime.cfg` or
-`addons.cfg` as described in the previous section.
-
-Navigate your browser to `http://<openhab server address>:8080` and select
-PaperUI from the list of UIs. Navigate to `Configuration -> System` on the left
-hand side and enable "Include Legacy 1.x Bindings" and "Access Remote
-Repositories". This will allow you to install your current bindings even if
-there is an openHAB 2 version available. Press "Save".
+Navigate your browser to `http://<openhab server address>:8080` and select PaperUI from the list of UIs. 
+Select "Expert" from the list of default packages. 
+Now select PaperUI and navigate to `Configuration -> System` on the left hand side.
+Enable "Include Legacy 1.x Bindings" and "Access Remote Repositories". 
+This will allow you to install your current bindings even if there is an openHAB 2 version available. 
+Press "Save".
 
 ![PaperUI System Settings](images/PaperUI-Config-System.png)
 
-Now select "Extensions" from the left and browse or search for each of the
-bindings and add-ons you currently use. Make note of those that do not appear in
-the list. Do not install any binding that has a version 2.0. If there is not a
-1.9.0 version we will install the binding manually (see below).
-
-This is a great opportunity to identify those bindings you actually use and
-only install those.
+Now select "Add-ons" from the left and browse or search for each of the bindings and add-ons you currently use. 
+Make note of those that do not appear in the list. 
+Do not install any binding that has a version 2.x at this time. 
+If there is not a 1.x.0 version we will install the binding manually (see below).
 
-When you find a binding to install, press the Install button and wait for
-installation to complete. Do not install the My.openHAB Binding yet.
+This is a great opportunity to identify those bindings you actually use and only install those.
 
-If you are running openHAB in a Docker container you need to create
-`<openHAB home>/userdata/persistence` manually prior to installing a persistence
-add-on.
+When you find a binding to install, press the Install button and wait for installation to complete. 
+Do not install the myopenhab/openhabcloud Binding yet.
 
-One thing to be aware of is any configuration done in PaperUI or Habmin gets
-saved to a non-human readable database. And since none of the administration
-UIs are currently capable of doing everything, you will end up with a mix of
-database and text based configuration. This is why we recommend not using
-PaperUI for those migrating to openHAB 2 from openHAB 1.x.
+One thing to be aware of is any configuration done in PaperUI gets saved to a files and folders under <openHAB 2 conf>.
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.